library/core/src/ops/function.rs

```diff
@@ -216,6 +216,10 @@ pub trait FnMut<Args>: FnOnce<Args> {
 )]
 #[fundamental] // so that regex can rely that `&str: !FnMut`
 #[must_use = "closures are lazy and do nothing unless called"]
+#[doc(alias = "(")]
+#[doc(alias = ")")]
+#[doc(alias = "()")]
+#[doc(alias = "( )")]
```

Comment:
These sholdn't be on closures; they're already on unit.

Reply:
I was thinking of function-call syntax, i.e. `f()` where `f: FnOnce`, similar to the `[]` aliases on the `Index` trait for supporting `v[]` for `v: Index`.

Reply:
I realize that, but that would make these ambiguous.

Reply:
I’m not sure what you mean by “ambiguous”. We already have `[]` on `slice`, `array`, `Index`, and `IndexMut`. We already have `()` on `unit` and `tuple`. I don’t see any additional ambiguity. Nonetheless, following the discussion in [this thread](https://github.com/rust-lang/rust/pull/88594#discussion_r702310581), a lot of changes, including the new alias for closures, are already removed from this PR, anyway.